pkg/agent/controller/networkpolicy/networkpolicy_controller.go

```diff
@@ -81,6 +81,24 @@ func NewNetworkPolicyController(antreaClient versioned.Interface,
 	return c
 }
 
+type NetworkPolicyInfoQuerier interface {
```

Comment:
maybe move the interface to pkg monitor as it's used there and there are two implementation in different packages so it's more fair there.
Also, see the general suggestion of where to put interfaces: https://github.com/golang/go/wiki/CodeReviewComments#interfaces

Reply:
Thanks for the suggestion. Have moved the interface to pkg/monitor/querier.go. Please help take a look.